docs/sources/aws_cloudwatch_log_group.md

```diff
@@ -38,17 +38,17 @@ partition "aws_cloudtrail_log" "cw_log_group_logs_prefix" {
   source "aws_cloudwatch_log_group" {
     connection        = connection.aws.default
     log_group_name    = "aws-cloudtrail-logs-123456789012-fd33b044"
-    log_stream_prefix = "456789012345_CloudTrail_us-east-1"
+    log_stream_names  = ["456789012345_CloudTrail_*"]
```

Comment:
This example is incorrect, we can add this as `Collect CloudTrail logs for a specific account and region`, and then have another example below this `Collect CloudTrail logs for all regions in an account` that uses wildcard